etcdctl/ctlv3/command/migrate_command.go

```diff
@@ -220,7 +220,7 @@ func writeKeys(w io.Writer, n *store.NodeExtern) uint64 {
 	if err != nil {
 		ExitWithError(ExitError, err)
 	}
-	fmt.Fprintf(w, string(b))
+	fmt.Fprintln(w, string(b))
```

Comment:
@hongchaodeng println adds a `\n` at the end of the line. why do we need that `\n`?